Dashboard.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function Dashboard() {
+  return <h1>Welcome to the Dashboard!</h1>;
```

Comment:
The Dashboard component is currently very basic and just returns a static header. It should be expanded to include the actual functionality of the dashboard.